uint8 trajectoryId
uint32 pieceOffset
TrajectoryPolynomialPiece[] pieces
---
